packages/tx-ingestion/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Hmm all of this boilerplate is unfortunate. This sort of thing makes me think this shouldn't be a monorepo....

Reply:
Instead use a single package & just split up the tests in such a way that we run only tests in a particular directory based on the env var (or something). Idk just feels like this current solution is pretty heavy weight for not much benefit